Tabby/components/FooterNavBar.tsx

```diff
@@ -1,15 +1,54 @@
-import { View } from "react-native";
-import { Link } from "expo-router";
-import React from "react";
-
-const FooterNavBar = () => {
-
-    return (
-        <View className=" flex-1 flex-row items-end">
-            <Link href="/library" className="p-5 text-white"> Library</Link>
-            <Link href="/recommendations" className="p-5 text-white"> Reccomendations</Link>
-        </View>
-    )
-}
-
-export default FooterNavBar
\ No newline at end of file
+import { View, Image, Text } from "react-native";
+import { Link, usePathname } from "expo-router";
+import React from "react";
+
+const FooterNavBar = () => {
+  const pathname = usePathname();
+  return (
+    <View className="absolute justify-center bottom-0 flex-row items-start h-16 w-screen bg-[#1d232b]">
+      <View className="absolute left-0 flex-col">

```

Comment:
A few spots in your code you use the "absolute" tag. This tag should be used as seldom as possible as it can likely cause conflicting UI with different devices.